### ISSUE TITLE
Actualize Feeds : Error invoking actualize_scipt.php with nginx on Synology DSM 6.x

### ISSUE THREAD

**Issue by Ribero** [FreshRSS/FreshRSS#2587]
### Context

FreshRSS instance is running on a Synology with the following configuration

- PHP 7.3
- MariaDB 10
- NGINX managed by WebStation
- Multiple Virtual Host in nginx with one dedicated to FreshRSS

Other NGINX virtual servers running on this Synology NAS (Fever, CRM, few other webapps...).
All Virtual Hosts are configured to use, each, a specific port number (same URL for all except the port number changing) and were all configured using the standard Synology Web Station GUI.

FreshRSS Installation went very smooth (all preflight check box are green/ok) and everything seems to works as expected from the end user Web GUI but trying to setup an auto-update of feeds of all FreshRSS users fails.

### Issue

Using

`php /volume1/web/freshrss/app/actualize_script.php
`

results in:

```
Starting feed actualization at 2019-10-20T16:00:37+02:00
FreshRSS install failed requirements:
• pdo-mysql
• fileinfo
• data
• cache
• users
• favicons
• ok
```

... and no feed actualization

As stated by @Alkarex in https://github.com/FreshRSS/FreshRSS/issues/2582#issuecomment-544267113 it is probably due to the fact that there is typically more than one PHP configuration on Synology, and the one for the cron might not be the same one than for the Web.

The proposed solution in https://github.com/FreshRSS/FreshRSS/issues/1853 does not work anymore as in DSM 6.2.2-24922 Update 3 (latest version) no more php56 command.

The default 'php' command is PHP Version 5.6.11 but fails:

```
FreshRSS install failed requirements:
• pdo-mysql
• fileinfo
• ok

```

When trying with `sudo php73 /volume1/web/freshrss/app/actualize_script.php` :

```
Starting feed actualization at 2019-10-20T19:16:48+02:00
Actualize admin...
PHP Fatal error:  Uncaught Minz_PDOConnectionException: Access to database is denied for `freshrss` (`mysql:host=localhost;dbname=freshrss;charset=utf8mb4`) in /volume1/web/freshrss/lib/Minz/ModelPdo.php:95
Stack trace:
#0 /volume1/web/freshrss/app/actualize_script.php(61): Minz_ModelPdo->__construct('admin')
#1 {main}
  thrown in /volume1/web/freshrss/lib/Minz/ModelPdo.php on line 95
Results: 
```

Of course FreshRSS work normally from the web gui (so no issue to access its database).

I opened this issue to record it for other users that eventually are having problems doing auto-update on Synology systems using nginx.

**Comment by Alkarex** [comment]
Please see this related issue and solution https://github.com/FreshRSS/FreshRSS/issues/2394#issuecomment-489428883

**Comment by Ribero** [comment]
Thanks for the link. However, my personal approach is to avoid any change that is "server wide". I'm afraid that the modification of the main and global `php.ini` could affect all other apps that are running well on the server with current php settings.

Anyway, the solution that finally works, for me, was I suspected, to replace localhost by `127.0.0.1` in `./data/config.php`

In resume, I replaced in `./data/config.php`: 

    'host' => 'localhost',

by

    'host' => '127.0.0.1',

Then update feeds using:

`sudo php73 /volume1/web/freshrss/app/actualize_script.php`

I do not know why the database is not reachable by hostname `localhost` in the Synology environment. May be it is related to the cluster configuration of my setup.
